### PR TITLE
Deprecation notice and az cli command suggestions

### PR DESCRIPTION
**Pull request by pierreca**
**iothub-explorer** will be retired on November 31st, 2018 and has been replaced by the [Azure CLI IoT Extension](https://aka.ms/iotcli).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,29 +1,5 @@
...
-
-As far as code is concerned the code associated with the PR will be pulled and run through the gated build before it makes it to the master branch. As much as possible, we insist on having tests associated with the code, and if necessary, additions/modifications to the requirement documents. As a matter of fact, the build will fail if code coverage goes down.
-
-Also, have you signed the [Contribution License Agreement](https://cla.microsoft.com/) ([CLA](https://cla.microsoft.com/))? A friendly bot will remind you about it when you submit your pull-request.
-
-If you feel like your contribution is going to be a major effort, you should probably give us a heads-up. We have a lot of items captured in our backlog and we release every two weeks, so before you spend the time, just check with us to make
-sure your plans and ours are in sync :) Just open an issue on github and tag it "enhancement" or "feature request"
+We will not address any issue regarding iothub-explorer except those related to the migration to the azure cli.
```
Comment (jongio) [inline .github/CONTRIBUTING.md]: Capitalize Azure CLI

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,23 +1,6 @@
 <!--
-Hi there! thank you for discovering and submitting an issue!
+Hi there!
+iothub-explorer will be retired on November 31st, 2018 and has been replaced by the [Azure CLI IoT Extension](https://aka.ms/iotcli).
 
-Please first tell us a little bit about the environment you're running:
-The commands in the comments can be run directly in a command prompt.
+We will not address any issue regarding iothub-explorer except those related to the migration to the azure cli.
```
Comment (jongio) [inline .github/ISSUE_TEMPLATE.md]: Capitalize Azure CLI

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,17 +1,6 @@
...
-- [ ] I have read the [contribution guidelines] (https://github.com/Azure/iothub-explorer/blob/master/.github/CONTRIBUTING.md).
-
-# Reference/Link to the issue solved with this PR (if any)
-
-# Description of the problem
-<!-- Please be as precise as possible: what issue you experienced, how often... -->
-
-# Description of the solution
-<!-- How you solved the issue and the other things you considered and maybe rejected --> 
+We will not address any pull-request regarding iothub-explorer except those related to the migration to the azure cli.
```
Comment (jongio) [inline .github/PULL_REQUEST_TEMPLATE.md]: Capitalize Azure CLI

File: iothub-explorer-monitor-uploads.js [hunk 4]
```diff
@@ -16,6 +16,9 @@ var getSas = require('./common.js').getSas;
 
 // Azure Event Hubs dependencies
 var ServiceClient = require('azure-iothub').Client;
+var showDeprecationText = require('./common.js').showDeprecationText;
+
+showDeprecationText('There is no equivalent command in the azure cli - if this feature is important to you please open an issue on the Azure CLI IoT Extension repository (https://aka.ms/iotcli)');
```
Comment (digimaun) [inline iothub-explorer-monitor-uploads.js]: change azure cli to Azure CLI for consistency

File: iothub-explorer-monitor-ops.js [hunk 5]
```diff
@@ -14,6 +14,9 @@ var serviceError = require('./common.js').serviceError;
 
 // Azure Event Hubs dependencies
 var EventHubsClient = require('azure-event-hubs').Client;
+var showDeprecationText = require('./common.js').showDeprecationText;
+
+showDeprecationText('There is no equivalent command in the azure cli - this feature is deprecated and replaced with Azure Monitor');
```
Comment (digimaun) [inline iothub-explorer-monitor-ops.js]: change azure cli to Azure CLI for consistency

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (digimaun) [reply .github/CONTRIBUTING.md]: consider replacing text with 'We will not address any issues regarding iothub-explorer except for those related to migrating towards the Azure CLI.'

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (digimaun) [inline .github/ISSUE_TEMPLATE.md]: consider replacing text with 'We will not address any issues regarding iothub-explorer except for those related to migrating towards the Azure CLI.'

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (digimaun) [reply .github/PULL_REQUEST_TEMPLATE.md]: consider replacing text with 'We will not address any pull-requests against iothub-explorer except for those related to migrating towards the Azure CLI.'

**Review by digimaun — CHANGES_REQUESTED** [review]
suggestions for updating sentences and consistency in Azure CLI format

**Review by digimaun — APPROVED** [review]

**Pull request merged**